crypto_kem/babybear/clean/Makefile

```diff
@@ -0,0 +1,21 @@
+# This Makefile can be used with GNU Make or BSD Make
+
+LIB=libbabybear_clean.a
+
+HEADERS = api.h melas_fec.h params.h ring.h threebears.h
+OBJECTS = kem.o melas_fec.o ring.o threebears.o 
+
+
+CFLAGS=-O3 -Wcast-align=strict -Wall -Wconversion -Wextra -Wpedantic -Wvla -Wmissing-prototypes -Wredundant-decls -std=c99 -I../../../common $(EXTRAFLAGS)
```

Comment:
@thomwiggers, does this mean that -Wcast-align=strict is now supported by all of our targets?
If yes, then we should add this to all the implementations.

Reply:
I don't think it is; there's still false positives probably.

Reply:
I think `-Werror` is missing here